lstchain/analysis/post_dl2.py

```diff
@@ -78,8 +78,8 @@ def analyze_wobble(config):
         n_off += np.sum(named_datasets[-1][1] < theta2_cut)
 
     stat = WStatCountsStatistic(n_on, n_off, 1./(n_points - 1))
-    lima_significance = stat.significance.item()
-    lima_excess = stat.excess
+    lima_significance = stat.sqrt_ts.item()
```

Comment:
What motivates this? Can you please explain? You change here the definition of excess significance from Li&Ma to sqrt(S/B)...

Reply:
That problem comes from the version of gammapy. The new version of gammapy requested for this new version of lstchain does not have those attributes. I updated them
https://docs.gammapy.org/dev/api/gammapy.stats.WStatCountsStatistic.html

Reply:
Can you link the relevant discussion please? Also a comment in the code with a reference to implementation is highly appreciated

Reply:
Which discussion? The code, implemented in the new version of lstchain, does not work. That solves the problem, it is only an attribute problem. I do not understand what else to do, sorry. What should we do in this case?

Reply:
ah, so this is just an API (method signature) change? If so, please just add a note about it in the code with a link to gammapy doc as you pointed out above.

Reply:
OK done. Thanks